src/assets/release-notes-1.26.0-alpha.1.json

```diff
@@ -0,0 +1,1003 @@
+{
+  "108832": {
+    "commit": "17dd76f5d49848e0bcb057035cda7946b578584f",
+    "text": "fix relative cpu priority for pods where containers explicitly request zero cpu by giving the lowest priority instead of falling back to the cpu limit to avoid possible cpu starvation of other pods",
+    "markdown": "Fix relative cpu priority for pods where containers explicitly request zero cpu by giving the lowest priority instead of falling back to the cpu limit to avoid possible cpu starvation of other pods ([#108832](https://github.com/kubernetes/kubernetes/pull/108832), [@waynepeking348](https://github.com/waynepeking348)) [SIG Node]",
+    "author": "waynepeking348",
+    "author_url": "https://github.com/waynepeking348",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/108832",
+    "pr_number": 108832,
+    "areas": ["kubelet"],
+    "kinds": ["bug"],
+    "sigs": ["node"]
+  },
+  "110695": {
```

Comment:
![image](https://user-images.githubusercontent.com/1404867/192447045-2f318ec6-27c1-4b6a-b774-04a20f509a5a.png)

This commit adds an entry like this. I don't think this should be picked into the release note at all. It has a `release-note-none` label in it.

Reply:
Good catch! Let me remove those _None_ entries